pkg/stream/enviroment.go

```diff
@@ -47,11 +49,18 @@ func NewEnvironment(options *EnvironmentOptions) (*Environment, error) {
 			if err != nil {
 				return nil, err
 			}
+			parameter.Scheme = u.Scheme
 			parameter.User = u.User.Username()
 			parameter.Password, _ = u.User.Password()
 			parameter.Host = u.Host
 			parameter.Port = u.Port()
-			parameter.Scheme = u.Scheme
+
+			if vhost := strings.TrimPrefix(u.Path, "/"); len(vhost) > 0 {
```

Comment:
Thank you.
Can you please add a test [here](https://github.com/rabbitmq/rabbitmq-stream-go-client/blob/main/pkg/stream/enviroment_test.go#L176).

maybe change the `"Environment Authentication"` to   `"Environment Authentication/Validation"`

Reply:
Sure! Thanks for the pointers.

Any idea on how to test that a valid vhost is able to be parsed/connect? Is it OK to actually create a vhost on the test rabbitmq?

Reply:
>Is it OK to actually create a vhost on the test rabbitmq?

Yes, actually I have to add some HTTP call to the management UI to complete the tests. 
[Here](https://github.com/rabbitmq/rabbitmq-stream-go-client/blob/main/examples/haProducer/http/http.go) there is an example.

If you want, you could add some basic HTTP call on the test suite starting from my example.

Thank you

Reply:
Thanks for the tip, calling from the HTTP API of the management server was definitely easier :)
Just pushed a commit with the new test.